src/Test/StateMachine/Internal/Parallel.hs

```diff
@@ -254,6 +256,24 @@ toForkOfOps h = Fork (mkOps l) p' (mkOps r)
     dynResp = fromMaybe (error "toForkOfOps: impossible.") . fromDynamic
   mkOps _  = error "mkOps: Impossible."
 
+toBoxDrawings :: forall cmd. (IxFunctor cmd, ShowCmd cmd, HasResponse cmd) => History cmd -> Doc
```

Comment:
`History` and `Operation` are defined in this module so we would get a cyclic import if I move stuff that depends on that from here.